apps-rendering/src/components/MainMedia/MainMediaImage/index.tsx

```diff
@@ -23,12 +24,16 @@ interface Props {
 	format: ArticleFormat;
 }
 
-const MainMediaImage: FC<Props> = ({ className, image, format }: Props) => {
+const MainMediaImage: FC<Props> = ({ image, format }: Props) => {
 	if (format.display === ArticleDisplay.Immersive) {
 		return <ImmersiveMainMediaImage image={image} format={format} />;
 	}
 
 	switch (format.design) {
+		case ArticleDesign.LiveBlog:
+			return <ImmersiveMainMediaImage image={image} format={format} />;
```

Comment:
I don't think we want to use the immersive main media for liveblogs?

```suggestion
```

Reply:
Yeah that's a really weird one. Not sure how that happened, I think it might have been a bad merge. Fixed now, whatever happened